test_scipts.py

```diff
@@ -66,3 +70,24 @@ def test_comment_fields():
   response = requests.get(f'https://jsonplaceholder.typicode.com/comments/{commentid}')
   assert expected_fields == set(response.json().keys())
 
+def test_create_new_comment():
+  new_comment = {"postId": 101, "id": 501, "name": "Andries Fokkema", "email": "andries_fokkema@epam.com", "body": "This is a new comment"}
+  response = requests.post('https://jsonplaceholder.typicode.com/comments/', new_comment)
+  assert isinstance(response.json()['id'], int)
+  assert response.json()["id"] == new_comment["id"]
+  assert response.json()["postId"] == new_comment["postId"]
+  assert response.json()["name"] == new_comment["name"]
+  assert response.json()["email"] == new_comment["email"]
+  assert response.json()["body"] == new_comment["body"]
```

Comment:
to remove unnecessary `.json()` in each line. We can simply assign result of `.json()` operation to variable:
`response_comment = response.json()`
  `assert response_comment["id"] == new_comment["id"]`
  `assert response_comment["postId"] == new_comment["postId"]`
  `assert response_comment["name"] == new_comment["name"]`
  `assert response_comment["email"] == new_comment["email"]`
  `assert response_comment["body"] == new_comment["body"]`